packages/integration_test/example/test_driver/extended_integration_test.dart

```diff
@@ -2,15 +2,60 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:io';
+import 'dart:typed_data';
+
+import 'package:collection/collection.dart';
 import 'package:flutter_driver/flutter_driver.dart';
 import 'package:integration_test/integration_test_driver_extended.dart';
 
 Future<void> main() async {
   final FlutterDriver driver = await FlutterDriver.connect();
   await integrationDriver(
     driver: driver,
-    onScreenshot: (String screenshotName, List<int> screenshotBytes) async {
-      // Return false if the screenshot is invalid.
+    onScreenshot: (
+      String screenshotName,
+      List<int> screenshotBytes, [
+      Map<String, Object?>? args,
+    ]) async {
+      // Here is an example of what updating a golden image might look like.
```

Comment:
@yjbanov @Piinks would it be acceptable to land this PR with the provided example in comments so that .png's are never added to this example? This still give some guidance or inspiration into how someone could hook up golden image testing if they weren't able to use Skia Gold.

Reply:
Optionally, I can also remove this example and just leave the changes in this PR that are outside the `example/` directory. We will do something like this for DevTools, since it looks like Skia Gold has not been hooked up yet for web integration tests (https://github.com/flutter/flutter/issues/86120). I'd like to unblock screenshot testing in integration tests for DevTools (even without skia gold support), so let me know what you think. Thanks.